File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.cpp
class Solution {
public:
    int maximumCount(vector<int>& nums) {
        int firstNonNeg = lower_bound(nums.begin(), nums.end(), 0) - nums.begin();
        int firstPos = lower_bound(nums.begin(), nums.end(), 1) - nums.begin();
        int neg = firstNonNeg, pos = nums.size() - firstPos;
        return max(pos, neg);
    }
};

File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.py
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        i = neg = pos = 0
        n = len(nums)
        while i < n and nums[i] < 0:
            i += 1
            neg += 1
        while i < n and nums[i] == 0:
            i += 1
        while i < n and nums[i] > 0:
            i += 1
            pos += 1
        return max(neg, pos)